src/Core/Domain/ServiceModel.ts

```diff
@@ -7,13 +7,13 @@ export type AttributeModel = AttributeValidation & {
   type: string;
   description?: string;
   modifier: string;
-  default_value: string | null;
+  default_value: string | string[] | null;
   default_value_set: boolean;
 };
 
 export interface StateModel {
   deleted: boolean;
-  label?: "info" | "success" | "warning" | "danger";
+  label?: "info" | "success" | "warning" | "danger" | null;
```

Comment:
is there a reasoning behind adding the null possibility ?

Reply:
at the current mock taken from real world example few states were missing label property so I had to add it to cover whole range of possible "states"

Reply:
I'm a bit confused, don't we need to add the missing states then, instead of covering it with null?

Reply:
If there is a possibility that backend returned label as null then I think it should be covered by corresponding interface. At least that's how I understood that case, especially that there is "no label" possibility in the service Inventory/Details